index.css

```diff
@@ -189,22 +226,168 @@ li {
     text-justify: inter-word;
   }
 
-  #social-media {
+  .social-media {
     display: flex;
     flex-direction: column;
     margin-top: 60px;
+    gap: 10px;
   }
 
-  #social-media ul {
+  .social-media ul {
     display: flex;
     gap: 15px;
     margin-top: 10px;
   }
 
-  #social-para {
+  .social-para {
     text-decoration: none;
     font-style: normal;
     font-weight: 500;
     font-size: 16px;
   }
+
+  #projects-section {
+    background: #e5e5e5;
+  }
```

Comment:
I suggest we stay consistent with the color used for the body `#eee`, so our project sticks to the design😉
<img width="313" alt="color" src="https://user-images.githubusercontent.com/64643995/220057175-096b9cc3-001b-4e1c-aa53-984424acc6d0.png">